run/events-pubsub/app_test.py

```diff
@@ -0,0 +1,100 @@
+import ast
+
+import base64
+
+import unittest
```

Comment:
I'm sorry to say, but have you read the AUTHORING_GUIDE.md?
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/2b508e61ff8fb045d48f3c4a320e688bb0b523ad/AUTHORING_GUIDE.md#testing-guidelines

We usually use pytest for writing tests. Can you rewrite?

Reply:
Understood. My apologies I'm new to this community so I was unaware of the authoring guide. Would have saved me a lot of rewriting if I'd noticed it earlier!